docarray/documents/image.py

```diff
@@ -92,10 +95,24 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[ImageUrl]
-    tensor: Optional[ImageTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[ImageBytes]
+    url: Optional[ImageUrl] = Field(
+        description='URL of the remote image',
+        example='https://github.com/docarray/docarray/blob/main/tests/toydata/image-data/apple.png?raw=true',
+    )
+    tensor: Optional[ImageTensor] = Field(
+        description='''Tensor object of the image which can be specifed to one of
+    `ImageNdArray`, `ImageTorchTensor`, `ImageTensorflowTensor`.''',
+        example='img.url.load()',
```

Comment:
let's remove this if we cannot show. Think about the example are something u can pass in the `json` and be accepted by a REST service.

Reply:
Updated PR with suggestions